WPFHospitalEditor/Model/EquipmentUnit.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace WPFHospitalEditor.Model
+{
+    public class EquipmentUnit : Entity<int>
+    {
+
+        [ForeignKey("CurrentLocation")]
+        public int? CurrentLocationId { get; set; }
+        public Room CurrentLocation { get; set; }
```

Comment:
Is this question mark here by mistake? Not sure if foreign key can be null.

Reply:
Fixed.